docs/spelling/allowed_words.txt

```diff
@@ -85,6 +85,7 @@ URI
 URIs
 VM
 VPN
+VSCode's
```

Comment:
is `'s` needed here? 
```suggestion
VSCode
```

Reply:
yes lol — blame whatever spelling lib we're using